### PR TITLE
Fix async calling of down and up; fixes #11

### LINKED ISSUE #11: Errors due to parallel docker-compose up/down in compose.bat
Using `START /B` in `compose.bat` leads to parallel execution of `docker-compose down` and  `docker-compose up` in
operations like `Up:`, `Reboot:`, ...

With certain timing conditions observed error cases include:
- untimely removal of network during/after container startup
  - container can not find network
  - trying to remove network while in use by containers
  - WebService unreachable
- untimely removal of `database` container while `webservice` is still initializing/running
- skipping `up` due to containers already running and only bringing them down instead

There may also be other effects/combinations.

If there is a reason to use `START /B` at all, it should be changed to `START /WAIT /B`.
Or dropped altogether.
Sequential calls to `docker-compose` are also used in the `compose.sh` script.

Synchronous invocation would in my opinion also improve command line output for `compose.bat` in case of single `docker-compose` operations.

### PR DESCRIPTION
**Pull request by alexatrhea**
`latest` images also used now

### CODE REVIEW

**Comment by as-marcb** [comment]
Keeping the 2nd calls async still interferes with console prompt output and makes any potential command chaining (call from other script) problematic.

Test* for some reason still use "START" instead of "CALL" (should be cosmetic only).
Effects of CALL and START (without /B) are to my knowledge identical for (MZ-)Binaries (and even using nothing at all should be fine).

For me the pattern
```batch
:Rc
docker-compose.exe -f docker-compose-rc.yml down --remove-orphans
docker-compose.exe -f docker-compose-rc.yml up -d
GOTO End
```
has the same effect on `docker.compose` execution and due to the missing '/B' the script actually returns after the 2nd command is **finished**.

File: compose.bat [hunk 1]
```diff
@@ -28,25 +28,25 @@ START /B docker-compose start
...
 
 :Stp
-START /B docker-compose stop
+CALL docker-compose stop
 GOTO End
 
 :Up
-START /B docker-compose down --remove-orphans
+CALL docker-compose down --remove-orphans
 START /B docker-compose up -d
```
Comment (as-marcb) [inline compose.bat:36]: Script still returns before `up` is completed.

Interferes wit command prompt and may have issues for command chaining (called from different script)

File: compose.bat [hunk 2]
```diff
@@ -55,21 +55,21 @@ START /B docker-compose -f docker-compose-rc.yml down --remove-orphans
...
 
 :TestRc
-START /B docker-compose -f docker-compose-rc-test.yml down --remove-orphans
+CALL docker-compose -f docker-compose-rc-test.yml down --remove-orphans
 START /B docker-compose -f docker-compose-rc-test.yml up -d
 GOTO End
 
 :TestRcDown
-START /B docker-compose -f docker-compose-rc-test.yml down --remove-orphans
+START docker-compose -f docker-compose-rc-test.yml down --remove-orphans
```
Comment (as-marcb) [inline compose.bat:63]: `CALL` is used in similar cases above

**Comment by alexatrhea** [comment]
> Keeping the 2nd calls async still interferes with console prompt output and makes any potential command chaining (call from other script) problematic.
> 
> Test* for some reason still uses "START" instead of "CALL" (should be cosmetic only).
> Effects of CALL and START (without /B) are to my knowledge identical for (MZ-)Binaries (and even using nothing at all should be fine).
> 
> For me the pattern
> 
> ```batchfile
> :Rc
> docker-compose.exe -f docker-compose-rc.yml down --remove-orphans
> docker-compose.exe -f docker-compose-rc.yml up -d
> GOTO End
> ```
> 
> has the same effect on `docker-compose` execution and due to the missing asynchronous execution the script actually returns after the 2nd command is **finished**.

the reason for the async call to the UP is that closing the console (or Ctrl+C) after calling does not stop the containers

**Review by as-marcb — APPROVED** [review]
Async on 2 invocations

**Review by as-marcb — APPROVED** [review]
Has now same synchronous behavior as `.sh` version.
Console output now also in expected order.

**Comment by alexatrhea** [comment]
> Has now same synchronous behavior as `.sh` version.
> Console output now also in expected order.

Thanks for your help!

**Pull request merged**